slemicro/get_kubeconfig.sh

```diff
@@ -118,6 +123,10 @@ else
 	curl -sk "https://${RANCHERHOSTNAME}/v3/clusters/local?action=generateKubeconfig" -X 'POST' -H 'content-type: application/json' -H "Authorization: Bearer ${TOKEN}" | jq -r .config > ${TMPKUBECONFIG} 
 fi
 
-echo ${TMPKUBECONFIG}
-
-exit 0
+if [ -z "${OUTPUT_FILE}" ]; then
+  cat echo ${TMPKUBECONFIG}
```

Comment:
Shouldn't this be just 'cat ${TMPKUBECONFIG}' ?

Reply:
oops, yes! 😂

Reply:
Fixed